markdown/events/2021/bytesize-28-nf-core-sarek.md

```diff
@@ -27,3 +27,121 @@ It is our hope that these talks / videos will build an archive of training mater
 This week, José Fernández Navarro ([@jfnavarro](https://github.com/jfnavarro)) will tell us all about the nf-core/sarek pipeline.
 
 nfcore/sarek is a workflow designed to detect variants on whole genome or targeted sequencing data. Initially designed for Human, and Mouse, it can work on any species with a reference genome. Sarek can also handle tumour / normal pairs and could include additional relapses.
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: The content has been edited for reader-friendliness**
+
+[0:01](https://youtu.be/6EIGUe5sjNo&t=1)
+(host) Hi, Maxime here. I'm glad today to welcome you again for another bytesize talk. Today it will be about the pipeline Sarek and I think it's a very good opportunity that we have one of our power users who will be presenting this pipeline. In the end, it would be so boring if I were to present that. I would like to thank again the Chan Zuckerberg Initiative to help us do this bytesize talk every week. Right now, let's head over over to you, José to present us Sarek.
+
+[0:48](https://youtu.be/6EIGUe5sjNo&t=48)
+Thanks, Maxime. Hi, everyone. I would like to start by thanking the nf-core and the Sarek people for allowing me to give this talk. As Maxime said, I'm one of the power users and now also contributor. I would also like to thank the nf-core community in general. I am an active user of other pipelines and I'm always present in Slack and I'm very happy of this initiative and how the community is growing and they're setting up now the standards for pipelines. I'm really thankful. Well, this talk has been live streamed and recorded on YouTube. I like to say hi to everyone that is out there watching or will be watching later on.
+
+[1:38](https://youtu.be/6EIGUe5sjNo&t=98)
+A little bit about me, I guess this is a mandatory slide, but I'm not gonna talk too much about me because this is about Sarek, it's not about me. I have a bachelor degree in computer science, a master in computational biology and a PhD in computational biology. You want to check something about my research or my projects, you can see my GitHub or my research gate page. Some facts about me, I'm born and raised in Spain. The picture there, right there is my hometown this summer. But I did most of my studies and my career in Sweden, Stockholm, and I got to meet Maxime and Phil and some nf-core people there. I'm one of the regional developers of Spatial Transcriptomics, which is a technology that you might have heard of. So I accumulated a bunch of years experience both in academia and industry. I was even group leader of a bioinformatics unit at some point, but I decided to change careers and move to industry. Now I'm senior consultant at BioLizard, which is a consultancy company that specialize in bioinformatics. But that's enough about me.
```

Comment:
```suggestion
A little bit about me, I guess this is a mandatory slide, but I'm not gonna talk too much about me because this is about Sarek, it's not about me. I have a bachelor degree in computer science, a master in computational biology and a PhD in computational biology. You want to check something about my research or my projects, you can see my GitHub or my ResearchGate page. Some facts about me, I'm born and raised in Spain. The picture there, right there is my hometown this summer. But I did most of my studies and my career in Sweden, Stockholm, and I got to meet Maxime and Phil and some nf-core people there. I'm one of the regional developers of Spatial Transcriptomics, which is a technology that you might have heard of. So I accumulated a bunch of years experience both in academia and industry. I was even group leader of a bioinformatics unit at some point, but I decided to change careers and move to industry. Now I'm senior consultant at BioLizard, which is a consultancy company that specialize in bioinformatics. But that's enough about me.
```